generative_ai/model_tuning/supervised_advanced_example.py

```diff
@@ -31,6 +31,12 @@ def gemini_tuning_advanced() -> sft.SupervisedTuningJob:
     # PROJECT_ID = "your-project-id"
     vertexai.init(project=PROJECT_ID, location="us-central1")
 
+    # TODO(developer): If you need to use BYOSA, Update and un-comment the below line
+    # vertexai.init(service_account="your-service-account")
+
+    # TODO(developer): If you need to use CMEK, Update and un-comment the below line
+    # vertexai.init(encryption_spec_key_name="your-kms-key")
+
```

Comment:
Approving the PR but, what do you think of something like the following to provide a little more context to folks who are seeing these samples and might not know what BYOSA or CMEK are?

# Initialize Vertex AI with your service account for BYOSA (Bring Your Own Service Account).
# Uncomment the following and replace "your-service-account" 
# vertexai.init(service_account="your-service-account")


# Initialize Vertex AI with your CMEK (Customer-Managed Encryption Key).
# Un-comment the following line and replace "your-kms-key" 
# vertexai.init(encryption_spec_key_name="your-kms-key")